content/200-orm/200-prisma-client/600-observability-and-logging/250-opentelemetry-tracing.mdx

```diff
@@ -99,15 +99,17 @@ generator client {
 }
 ```
 
-### Step 3: Install OpenTelemetry packages
+</details>
 
-Finally, install the appropriate OpenTelemetry packages, as follows:
+### Step 2: Install OpenTelemetry packages
+
+Now install the appropriate OpenTelemetry packages, as follows:
 
 ```console
```

Comment:
FWIW I added this thing today: https://github.com/prisma/prisma/pull/25878, so now users don't have to install the `@opentelemetry/instrumentation` package (unless they need it for whatever reason separately) and can do

```ts
import { PrismaInstrumentation, registerInstrumentations } from '@prisma/instrumentation'
```

instead of

```ts
import { registerInstrumentations } from '@opentelemetry/instrumentation'
import { PrismaInstrumentation } from '@prisma/instrumentation'
```

For us, compatibility with different `@opentelemetry/instrumentation` versions is less of an issue if most users use our re-export, so perhaps it's worth to use it in the docs.